src/components/general/Speakers.vue

```diff
@@ -102,25 +104,51 @@ export default {
 	position: relative;
 }
 
-.image {
+.image-wrapper {
 	position: relative;
 	width: calc(100% - var(--icon-width) - .5rem);
 	height: 100%;
+	overflow: hidden;
 }
-.image>img {
-	width: 100%;
-	height: 100%;
-	object-fit: cover;
-	border-radius: var(--tile-radius);
-}
-.image-background {
+.image-wrapper>.positioning {
 	position: absolute;
 	left: 0;
 	bottom: 0;
-	z-index: -1;
 	width: 100%;
 	height: 80%;
+}
+.image-background {
 	border-radius: var(--tile-radius);
+	z-index: -1;
+	transition: var(--bio-transition);
+}
+.image-speaker {
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+	border-radius: var(--tile-radius);
+	transition: var(--bio-transition);
+}
+
+.bio {
+	box-sizing: border-box;
+	padding: 1rem;
+	z-index: 2;
+
+	opacity: 0;
+	color: whitesmoke;
+	transition: var(--bio-transition);
+}
+.bio:hover{
+	opacity: 100%;
+}
+.bio:hover ~ .image-speaker {
+	/* filter: invert(51%) sepia(61%) saturate(3831%) hue-rotate(162deg) brightness(96%) contrast(101%); */
+	/* filter: drop-shadow(0 -10px var(--img-color)); */
+	filter: brightness(0);
+}
+.bio:hover ~ .image-background {
+	background: black !important;
 }
```

Comment:
I realize this is my doing, but do you think you'd be able to change it from "on bio hover" to "on image-wrapper hover"? I don't think it would be too difficult.

The only reason I ask is because hover on the tops of people's heads doesn't active the transition current, and I think it should. What are your thoughts @liangel02 ?

Reply:
I agree, I think it should, I'll change that!